Resources/Patches/save/credits/strings.event

```diff
@@ -237,7 +237,7 @@ timeFoundCreditsEnglish:
 String("Found at:"); BYTE 0
 
 timeFoundCreditsFrench:
-String(""); BYTE 0
+String("Trouve en:"); BYTE 0
```

Comment:
Trouvé à: (closer to the actual meaning but "en" works too. Means Found in instead of Found at)

Reply:
Ok modifié